invenio_oauthclient/utils.py

```diff
@@ -244,3 +238,17 @@ def rebuild_access_tokens(old_key):
     """
     current_app.logger.info('rebuilding RemoteToken.access_token...')
     rebuild_encrypted_properties(old_key, RemoteToken, ['access_token'])
+
+
+def _get_csrf_disabled_param():
+    """Return the right param to disable CSRF depending on WTF-Form version.
+
+    From Flask-WTF 0.14.0, `csrf_enabled` param has been deprecated in favor of
+    `meta={csrf: True/False}`.
+    """
+    import flask_wtf
+    from pkg_resources import parse_version
+    supports_meta = parse_version(flask_wtf.__version__) >= parse_version(
+        "0.14.0")
+    return {'meta': {'csrf': False}} if supports_meta else \
```

Comment:
maybe keep the same format as on https://github.com/inveniosoftware/invenio-userprofiles/pull/86/files :)